tap-snapshots/test/tsconfig.ts.test.cjs

```diff
@@ -116,7 +116,6 @@ Object {
     "module": "nodenext",
     "moduleResolution": "nodenext",
     "rootDir": "../src",
-    "target": "es2022",
```

Comment:
Should we have tests for whether we preserve the `target` and when we don't?

Reply:
The [existing test case](https://github.com/isaacs/tshy/blob/79ae57078ba37acb45332b5699c4b0088d76650c/test/tsconfig.ts#L30) actually covers that case already, since it
- Generates everything from scratch, including the recommended tsconfig.json (which sets target to es2022)
- Then it makes a snapshot -- this is the snapshot you're commenting on, and the target field is not present, as expected with the new behavior
- Then the tsconfig.json is overwritten with a custom configuration, which omits the target field
- Finally it takes another snapshot, which is there is no diff for since target is still being set to es2022 as it used to be